test/components/views/right_panel/PinnedMessagesCard-test.tsx

```diff
@@ -224,7 +239,7 @@ describe("<PinnedMessagesCard />", () => {
             ...PollResponseEvent.from([answers[option].id], poll.getId()).serialize(),
             event: true,
             room: "!room:example.org",
-            user,
+            user: user as string,
```

Comment:
I'm mildly surprised it didn't figure this out, but if it works it works I guess. 

Other option might be to type the array, but that's a bit noisier.